app/assets/stylesheets/application_redesign.scss

```diff
@@ -132,4 +132,14 @@ summary .bi-chevron-right::before {
 label[for="user-header-hide"] {
   margin-left: -1.5em;
   margin-top: .6em;
+}
+
+input[type="radio"] {
+  width: 22px;
+  height: 22px;
+  margin-right: 4px;
+}
```

Comment:
Can we use the standard bootstrap styling for radio buttons here?
https://getbootstrap.com/docs/5.3/forms/checks-radios/#radios

I think/hope it's close (enough).

Reply:
Probably close enough, though I do think the radio button outline is a bit light, not sure if that is an accessibility problem. Automated checkers don't think so. Screenshots of before and after.
![Screenshot 2024-04-04 at 11 23 54 AM](https://github.com/sul-dlss/sul-requests/assets/19173991/1bc56e8f-7dd1-42aa-8f5c-3c098ffd9909)
![Screenshot 2024-04-04 at 11 36 40 AM](https://github.com/sul-dlss/sul-requests/assets/19173991/a04064a3-8e8a-4652-ac5b-217217eb099b)